cmd/nginx-ingress/main_test.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"bytes"
+	"regexp"
+	"testing"
+
+	nic_logger "github.com/nginxinc/kubernetes-ingress/internal/logger"
+	nic_glog "github.com/nginxinc/kubernetes-ingress/internal/logger/glog"
+)
+
+func TestLogFormats(t *testing.T) {
+	testCases := []struct {
+		name   string
+		format string
+		wantre string
+	}{
+		{
+			name:   "glog level format message",
```

Comment:
nit: a log format is not a level

Reply:
left over from similar test, will remove